python/aitemplate/backend/cuda/gemm_universal/gemm_rrr_bias.py

```diff
@@ -0,0 +1,327 @@
+#  Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+"""
+GEMM Specialization for
+C = GeMM(A, B) + bias
+where A[RowMajor][M, K], B[ColMajor][N, K], bias[RowMajor][N]
```

Comment:
Not: for `gemm_rrr_bias` should be `B[ColMajor][K, N]`.

Reply:
OK, thanks!